tools/Azure.Mcp.Tools.Sql/src/Commands/Database/DatabaseCreateCommand.cs

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine;
+using System.CommandLine.Parsing;
+using Azure.Mcp.Core.Commands;
+using Azure.Mcp.Core.Extensions;
+using Azure.Mcp.Core.Models.Command;
+using Azure.Mcp.Tools.Sql.Commands;
+using Azure.Mcp.Tools.Sql.Models;
+using Azure.Mcp.Tools.Sql.Options;
+using Azure.Mcp.Tools.Sql.Options.Database;
+using Azure.Mcp.Tools.Sql.Services;
+using Microsoft.Extensions.Logging;
+
+namespace Azure.Mcp.Tools.Sql.Commands.Database;
+
+public sealed class DatabaseCreateCommand(ILogger<DatabaseCreateCommand> logger)
+    : BaseDatabaseCommand<DatabaseCreateOptions>(logger)
+{
+    private const string CommandTitle = "Create SQL Database";
+
+    private readonly Option<string> _skuNameOption = SqlOptionDefinitions.SkuNameOption;
```

Comment:
Rebase and update to new Option design.  new-command.md can help you

Reply:
rebasing this noon